src/main/java/com/github/piedpiper/node/stepfunctions/StepFunctionsServiceNode.java

```diff
@@ -0,0 +1,54 @@
+package com.github.piedpiper.node.stepfunctions;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+
+import com.github.piedpiper.common.PiedPiperConstants;
+import com.github.piedpiper.node.BaseNode;
+import com.github.piedpiper.node.NodeInput;
+import com.github.piedpiper.node.NodeOutput;
+import com.github.piedpiper.node.ParameterMetadata;
+import com.google.common.collect.Lists;
+
+
+public class StepFunctionsServiceNode extends BaseNode {
```

Comment:
Make this a factory please. DO not extend this as base node here  as they will start appearing as separate nodes in Pied piper

Reply:
should this extend AWSNode?I know that we don't use it but please extend AWS Node irrespective. SO that long terms the structure remains right